packages/orchestration/src/proposals/orchestration-proposal.js

```diff
@@ -58,6 +62,24 @@ export const setupOrchestrationVat = async (
   orchestrationKitP.resolve(newOrchestrationKit);
 };
 
+/**
+ * @param {BootstrapPowers & {
+ *   installation: {
+ *     consume: {
+ *       stakeBld: Installation<
+ *         import('../../src/examples/stakeBld.contract.js').start
```

Comment:
that would be a `Promise<Installation<...>>`, right?

In fact, why declare a `stakeBld` installation here at all? it's not used.

Reply:
thanks. leftover from an earlier state. I'll remove